### DISCUSSION TITLE
Orientantions about wlan authentication modes

### DISCUSSION THREAD

**Discussion by AndryckSant** [micropython/micropython#13563] [ESP32] [answered]
I have found an interesting new situation while using my ESP32-WROVER-E with uPython.

While using the "wlan.scan()" I found someone's wi-fi that has an authmode equal to 7. I searched about it and found nothing.
I just reached out the old uPython Forum, where has a conversation some extra modes and the same information on the latest uPython version documentation.
![Captura de tela de 2024-01-30 14-35-48](https://github.com/micropython/micropython/assets/115589831/3803e16d-4997-4d47-a09a-1f9f8bb8ad28)
The printing order is: SSID, BSSID, Channel,Strength, Authmode.

The Wifi named "Ventania" has an authmode defined as '7'.

Does somebody know anything about it?

**Comment by ekspla** [comment, marked as answer]
Have a look at [the ESP-IDF document](https://docs.espressif.com/projects/esp-idf/en/latest/esp32/api-reference/network/esp_wifi.html#_CPPv416wifi_auth_mode_t).

**Comment by AndryckSant** [comment]
Thanks. I was looking for this specific information.
